nss-day-cohort-47/tabloidfullstack-bingo
# Hp edit category

# Description
Implemented edit functionality for category objects.

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Navigate to categories tab then select the edit affordance of a category, type a new name in the input field and select save. You should be redirected to the categories page and see the updated name of the edited category.
# Checklist:
